.rubocop_todo.yml

```diff
@@ -1,18 +1,103 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2015-10-08 14:05:42 -0700 using RuboCop version 0.34.2.
+# on 2023-03-17 10:29:23 UTC using RuboCop version 1.28.2.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
 # Offense count: 1
+# This cop supports safe auto-correction (--auto-correct).
+# Configuration parameters: Include.
+# Include: **/*.gemspec
+Gemspec/DateAssignment:
+  Exclude:
+    - 'metadata-json-lint.gemspec'
+
+# Offense count: 3
+# This cop supports safe auto-correction (--auto-correct).
+Lint/RedundantCopDisableDirective:
+  Exclude:
+    - 'Rakefile'
+    - 'lib/metadata-json-lint/schema.rb'
+    - 'spec/spec_helper.rb'
+
+# Offense count: 2
 Lint/RescueException:
   Exclude:
     - 'lib/metadata_json_lint.rb'
 
 # Offense count: 1
-# Configuration parameters: Exclude.
-Style/Documentation:
+# Configuration parameters: AllowComments, AllowNil.
+Lint/SuppressedException:
   Exclude:
+    - 'Rakefile'
```

Comment:
Can you fix this one? It's in the `github_changelog_generator` bit. There's also a redundant cop disable above it. If you add a comment in the `rescue` section that it's optional then you can drop this TODO.

Reply:
done